### PR TITLE
Normalize cloud-controller concept page

### PR DESCRIPTION
**Pull request by tengqm**
This PR wraps the long lines in the cloud-controller page where appropriate. It also removes some useless empty lines, fixes a numbered list by removing explicit numbering.

### CODE REVIEW

File: content/en/docs/concepts/architecture/cloud-controller.md [hunk 1]
```diff
@@ -206,12 +207,21 @@ rules:
...
-To upgrade a HA control plane to use the cloud controller manager, see [Migrate Replicated Control Plane To Use Cloud Controller Manager](/docs/tasks/administer-cluster/controller-manager-leader-migration/).
+To upgrade a HA control plane to use the cloud controller manager, see
+[Migrate Replicated Control Plane To Use Cloud Controller Manager](/docs/tasks/administer-cluster/controller-manager-leader-migration/).
 
 Want to know how to implement your own cloud controller manager, or extend an existing project?
 
-The cloud controller manager uses Go interfaces to allow implementations from any cloud to be plugged in. Specifically, it uses the `CloudProvider` interface defined in [`cloud.go`](https://github.com/kubernetes/cloud-provider/blob/release-1.21/cloud.go#L42-L69) from [kubernetes/cloud-provider](https://github.com/kubernetes/cloud-provider).
+The cloud controller manager uses Go interfaces to allow implementations from any cloud to be plugged in.
+Specifically, it uses the `CloudProvider` interface defined in
+[`cloud.go`](https://github.com/kubernetes/cloud-provider/blob/release-1.26/cloud.go#L43-L69) from
```
Comment (tengqm) [inline content/en/docs/concepts/architecture/cloud-controller.md:217]: To reviewers: This link is updated from 1.21 to 1.26.

File: content/en/docs/concepts/architecture/cloud-controller.md [hunk 2]
```diff
@@ -48,10 +46,10 @@ when new servers are created in your cloud infrastructure. The node controller o
 hosts running inside your tenancy with the cloud provider. The node controller performs the following functions:
 
 1. Update a Node object with the corresponding server's unique identifier obtained from the cloud provider API.
-2. Annotating and labelling the Node object with cloud-specific information, such as the region the node
+1. Annotating and labelling the Node object with cloud-specific information, such as the region the node
    is deployed into and the resources (CPU, memory, etc) that it has available.
-3. Obtain the node's hostname and network addresses.
-4. Verifying the node's health. In case a node becomes unresponsive, this controller checks with
+1. Obtain the node's hostname and network addresses.
```
Comment (sftim) [inline content/en/docs/concepts/architecture/cloud-controller.md:51]: We could also:
```suggestion
1. Obtaining the node's hostname and network addresses.
```

File: content/en/docs/concepts/architecture/cloud-controller.md [hunk 3]
```diff
@@ -107,7 +105,9 @@ routes appropriately. It requires Get access to Node objects.
 
 ### Service controller {#authorization-service-controller}
 
-The service controller listens to Service object Create, Update and Delete events and then configures Endpoints for those Services appropriately (for EndpointSlices, the kube-controller-manager manages these on demand).
+The service controller listens to Service object Create, Update and Delete events and then
```
Comment (sftim) [inline content/en/docs/concepts/architecture/cloud-controller.md:108]: Optional further cleanup:
```suggestion
The service controller watches for Service object **create**, **update** and **delete** requests,
and then
```

and an equivalent change everywhere else.

Within the Kubernetes API (eg RBAC types), these verbs are written all-lowercase. See https://kubernetes.io/docs/reference/using-api/api-concepts/ for other precedents.

File: content/en/docs/concepts/architecture/cloud-controller.md [hunk 4]
```diff
@@ -107,7 +105,9 @@ routes appropriately. It requires Get access to Node objects.
...
+configures Endpoints for those Services appropriately (for EndpointSlices, the
+kube-controller-manager manages these on demand).
```
Comment (sftim) [inline content/en/docs/concepts/architecture/cloud-controller.md:110]: Is this text still true given that we now favor EndpointSlices? I know that one can use Services even without a cloud provider integration.

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm

File: content/en/docs/concepts/architecture/cloud-controller.md (hunk 4 above)
Comment (tengqm) [reply content/en/docs/concepts/architecture/cloud-controller.md:110]: I think this is still true. cloud-controller-manager is not a must have.

File: content/en/docs/concepts/architecture/cloud-controller.md [hunk 5]
```diff
@@ -88,13 +86,13 @@ to read and modify Node objects.
...
 
-- Get
-- List
-- Create
-- Update
-- Patch
-- Watch
-- Delete
+- get
+- list
```
Comment (kbhawkey) [inline content/en/docs/concepts/architecture/cloud-controller.md:90]: Is it important to use lowercase for the method names?

Comment (tengqm) [reply content/en/docs/concepts/architecture/cloud-controller.md:90]: good point, at least this has to be consistent. will revise.

Comment (tengqm) [reply content/en/docs/concepts/architecture/cloud-controller.md:90]: Checked, lowercase is correct.
This PR is good to go as is.

**Pull request merged**